tests/test_export.py

```diff
@@ -8,27 +8,34 @@
 
 from urdf2webots.importer import convertUrdfContent, convertUrdfFile
 
-testDirectory = os.path.abspath(os.path.dirname(os.path.realpath(__file__)))
+rootDirectory = os.path.dirname(os.path.dirname(__file__)) + os.path.sep
```

Comment:
I would remove the `+ os.path.sep` to be consistent with other `*Directory` variables and add `+ os.path.sep` in the `replace` statement where `rootDirectory` is used or define a new variable `rootDirectoryWithFinalSeparator`.